frontend/server/src/main/java/org/pytorch/serve/util/ApiUtils.java

```diff
@@ -416,6 +416,13 @@ private static DescribeModelResponse createModelResponse(
             resp.addWorker(workerId, startTime, isRunning, gpuId, memory, pid, gpuUsage);
         }
 
+        DescribeModelResponse.JobQueueStatus jobQueueStatus =
+                new DescribeModelResponse.JobQueueStatus();
+        jobQueueStatus.setRemainingCapacity(model.getJobQueueRemainingCapacity());
+        jobQueueStatus.setPendingRequests(model.getPendingRequestsInJobQueue());
+        jobQueueStatus.setConsecutiveFailedRequests(model.getFailedInfReqs());
```

Comment:
Question: Does this mean these metrics were already being calculated and with this PR we are just exposing them?

Reply:
Yes, this data is already available. This PR exposes it via the `describe` API endpoint.